kvmagent/kvmagent/plugins/ovn.py

```
@@ -188,46 +197,85 @@ def start_ovn_service(self, req):
+        err1, ovn_remote = vsctl.getOvsExternalIdsConfig("ovn-remote")
+        err2, ovn_encap_ip = vsctl.getOvsExternalIdsConfig("ovn-encap-ip")
+        err3, ovn_encap_type = vsctl.getOvsExternalIdsConfig("ovn-encap-type")
+        err4, ovn_bridge_mappings = vsctl.getOvsExternalIdsConfig("ovn-bridge-mappings")
+        err5, hostname = vsctl.getOvsExternalIdsConfig("hostname")
+
+        if err1 or err2 or err3 or err4 or err5 or \
+                ovn_remote != cmd.ovnRemoteConnection or \
+                ovn_encap_ip != cmd.ovnEncapIP or \
+                ovn_encap_type != cmd.ovnEncapType or \
+                ovn_bridge_mappings != 'flat:{}'.format(cmd.brExName) or \
+                hostname != cmd.hostIp:
```

Pylint (3.3.7)

[refactor] Too many boolean expressions in if statement (10/5)

(R0916)